tools/util.h

```diff
@@ -0,0 +1,15 @@
+#ifndef COROSYNC_TOOLS_UTIL_H_DEFINED
+#define COROSYNC_TOOLS_UTIL_H_DEFINED
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+extern int              util_strtonum(const char *str, long long int min_val,
+    long long int max_val, long long int *res);
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif /* COROSYNC_COMMON_LIB_UTIL_H_DEFINED */
```

Comment:
Please change also this comment.

Reply:
Thanks for pointing this!
Changed both in this PR and needle one:)